### PR TITLE
📝 Adding some details to section Supporting a new cloud provider in Clouds

### PR DESCRIPTION
**Pull request by gaspardmoindrot**

### CODE REVIEW

File: docs/setup/clouds/index.md [hunk 1]
```diff
@@ -31,7 +31,34 @@ The currently supported cloud providers are:
...
-`srcs/runners_manager/vm_creation` containing a `CloudManager`
-object matching [CloudManager] interface.
+`srcs/runners_manager/vm_creation` containing a
+`CloudManager` object matching [CloudManager] interface.
+
+### Retrieving VM parameters
+
+To communicate with the application, the user will provide
+a lot of information in a `yml` configuration file, such as
+`image_id`, `instance_type`, `security_group_ids`, and so on.
```
Comment (tcarmet) [inline docs/setup/clouds/index.md:41]: Be less precise, as those names are specific to AWS. Per example, image, flavor, security settings, etc.

File: docs/setup/clouds/index.md [hunk 2]
```diff
@@ -31,7 +31,34 @@ The currently supported cloud providers are:
...
+During development, a `config_example.yml` file will be created
+to test the setup of the new Cloud Provider and to provide
+information about the VM. Next, to send this information to
+the new Cloud Provider, the `AwsConfigVmType` class will be
```
Comment (tcarmet) [inline docs/setup/clouds/index.md:45]: As we are not specifically into the aws section, you can mention `AwsConfigVmType` but use it as an example. Or mention the base class.

File: docs/setup/clouds/index.md [hunk 3]
```diff
@@ -31,7 +31,34 @@ The currently supported cloud providers are:
...
+To retrieve this information, we use
+`runner.vm_type.config["xxx"]` in the new Cloud Manager file.
+
+### How to launch the project for testing
+
+First, create a `.env` file and fill in the necessary
+environment variables. Then, create a `config_example.yml` file
+(examples are provided in the subsections depending on
+the Cloud Provider). Finally, launch the project with the command
+`docker-compose up --build`.
```
Comment (tcarmet) [inline docs/setup/clouds/index.md:62]: You might want to review the local setup section (can't remember the exact name). There should be a step by step guide on how to run it, maybe provide a link to this section instead?

Comment (gaspardmoindrot) [reply docs/setup/clouds/index.md:62]: You're right, I was just repeating myself in relation to the section that corresponds to local running, so I'm deleting this section

**Review by tcarmet — APPROVED** [review]

**Pull request merged**